tests/TestClassDatabaseChanges.php

```diff
@@ -108,7 +108,7 @@ public function testInstall(): array
 		$this->_pDbChanges->install();
 
 		remove_filter('query', [$this, 'saveCreateQuery'], 1);
-		$this->assertGreaterThanOrEqual(self::NUM_NEW_TABLES, count($this->_createQueries));
+		$this->assertLessThanOrEqual(self::NUM_NEW_TABLES, count($this->_createQueries));
```

Comment:
I don't understand how this can be correct

Reply:
because, in TestClassAdminViewController run install DBChanges in the first so _createQueries = 0

Reply:
Isn't `assertEquals` what we need here?